### PR TITLE
chore: remove sims

### PR DESCRIPTION
**Pull request by rootulp**
Resolves https://github.com/celestiaorg/celestia-app/issues/693

### CODE REVIEW

**Comment by rootulp** [comment]
Will close in favor of https://github.com/celestiaorg/celestia-app/pull/692

**Review by evan-forbes — COMMENTED** [review]
do we also need to remove something from the root makefile?

I think we can reopen this since it can be isolated.

**Comment by rootulp** [comment]
> do we also need to remove something from the root makefile?



Yes. Likely https://github.com/celestiaorg/celestia-app/blob/main/Makefile#L76-L81

File: Makefile [hunk 1]
```diff
@@ -73,14 +73,7 @@ lint:
...
 .PHONY: lint
 
-###############################################################################
-###                           Tests & Simulation                            ###
-###############################################################################
-# The below include contains the tools target.
-include contrib/devtools/Makefile
-include contrib/devtools/sims.mk
-
-test-all: check test-race test-cover
```
Comment (rootulp) [inline Makefile:83]: `check` was removed b/c it is not defined in any of the Makefiles

**Review by evan-forbes — APPROVED** [review]
:+1:

**Pull request merged**